src/components/Home.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="restaurants">
+    <div class="restaurants__img"></div>
+    <h1 class="restaurants__header">Najlepsze restauracje w Poznaniu</h1>
+    <ul class="restaurants__list">
+      <li
+        class="restaurants__item"
+        v-for="restaurant in restaurants"
+        :key="
+     restaurant.restaurant.id"
+      >
+        <a :href="restaurant.restaurant.events_url">{{restaurant.restaurant.name}}</a>
+        Ocena: {{restaurant.restaurant.user_rating.aggregate_rating}}
+        Kuchnia: {{restaurant.restaurant.cuisines}}
```

Comment:
The API returns its info in English. It looks unusual-at-best to see `Kuchnia: Polish` in such a list.